tests/test_bulk_update_or_create.py

```diff
@@ -485,7 +488,7 @@ def test_auto_now(self):
         self.assertEqual(2, AutoNowModel.objects.all().count())
 
         for instance in AutoNowModel.objects.all():
-            self.assertEqual(instance.updated, date.today())
+            self.assertEqual(instance.updated, datetime.now(pytz.utc).date())
```

Comment:
Duplicate: What issue does this fix? How is it connected to table name issue of this PR?